components/omega/doc/design/OmegaV1GoverningEqns.md

```diff
@@ -0,0 +1,464 @@
+ +\nabla \cdot \left( \int_{s_k^{bot}}^{s_k^{top}}   {\bf u} dp \right)
+ + \int_{s_k^{bot}}^{s_k^{top}}\frac{\partial}{\partial z} \left(    w \right) dp
+= \int_{s_k^{bot}}^{s_k^{top}} S^h dp
+$$ (layer-int-mass-R13App)
+
+tracer:
+
+$$
+\frac{\partial }{\partial t} \int_{s_k^{bot}}^{s_k^{top}}  \varphi  dp
+ + \nabla \cdot \left( \int_{s_k^{bot}}^{s_k^{top}}  \varphi {\bf u} dp  \right)
+ + \int_{s_k^{bot}}^{s_k^{top}} \frac{\partial}{\partial z} \left(   \varphi w \right) dp
+= \int_{s_k^{bot}}^{s_k^{top}} \left( D^\varphi + S^\varphi \right) dp
+$$ (layer-int-tracer-R13App)
+
+
+Rewriting the conservation equations in with these layer-averaged variables results in the final layered equations,
+
+$$
+\frac{\partial \mathbf{u}_k}{\partial t}
++ q_k\left(h\mathbf{u}_k^{\perp}\right) 
++ \nabla K_k
+= 
+- \frac{1}{\rho} \nabla_r p - \nabla_r \Phi
++ {\bf D}_k^u + {\bf F}_k^u
+$$ (layered-momentum)
+
+$$
+\frac{\partial h_k}{\partial t} + \nabla_r \cdot \left(h_k \mathbf{u}_k\right) + \omega_{bot} - \omega_{top}= S^h_k
+$$ (layered-mass)
+
+$$
+\frac{\partial h_k \varphi_k}{\partial t} + \nabla_r \cdot \left(h_k \mathbf{u}_k \varphi_k\right) 
++ \varphi\omega_{bot} - \varphi\omega_{top} 
+=  D^\varphi_k + S^\varphi_k 
+$$ (layered-tracer)
+
+### Discrete Equations
+
+The discretized versions of the governing equations are
+
+$$
+\frac{\partial u_e}{\partial t} + \left[ \frac{{\mathbf k} \cdot \nabla \times u_e +f_v}{[h_i]_v}\right]_e\left([h_i]_e u_e^{\perp}\right)
+= 
+- \frac{1}{\rho_i} \nabla_r p_i - \nabla_r \Phi_i
+- \nabla K_i + \nu_2 \nabla^2 u_e - \nu_4 \nabla^4 u_e + \mathcal{D}_e + \mathcal{F}_e
+$$ (discrete-momentum)
+
+$$
+\frac{\partial h_i}{\partial t} + \nabla \cdot \left([h_i]_e u_e\right) = 0,
+$$ (discrete-thickness)
+
+$$
+\frac{\partial h_i \varphi_i}{\partial t} + \nabla \cdot \left(u_e [h_i \varphi_i]_e \right) = \kappa_2 h_i \nabla^2 \varphi_i - \kappa_4 h_i \nabla^4 \varphi_i,
+$$ (discrete-tracer)
+
+where subscripts $i$, $e$, and $v$ indicate cell, edge, and vertex locations ($i$ was chosen for cell because $c$ and $e$ look similar).  Here square brackets $[\cdot]_e$ and $[\cdot]_v$ are quantities that are interpolated to edge and vertex locations. The interpolation is typically centered, but may vary by method, particularly for advection schemes. For vector quantities, $u_e$ denotes the normal component at the center of the edge, while $u_e^\perp$ denotes the tangential component. In the discrete system, the normal component $u_e$ points positively from the lower cell index to the higher cell index, while the tangential component $u_e^\perp$ points positively $90^o$ to the left of $u_e$ (for unit vectors, ${\mathbf n}_e^\perp = {\mathbf k}\times {\mathbf n}_e$).
+
+The discretized momentum drag and forcing terms are
+$$
+\mathcal{D}_e = - Ra \, u_e - C_D \frac{u_e\left|u_e\right|}{[h_i]_e}
+$$
+
+$$
+\mathcal{F}_e =  C_W \frac{(u_W - u_e)\left|u_W - u_e\right|}{[h_i]_e}
```

Comment:
since this comes from the coupler, why not just write as $$tau/(rho*[h_i]_3}$$ -- how will fluxes work?  is this h_i a pressure thickness?  units don't seem to worf if that is the case.

Reply:
Good point. I updated that and added a [wind forcing section](https://portal.nersc.gov/project/e3sm/mpeterse/omega/design-doc-v1-eqns/_build/html/design/OmegaV1GoverningEqns.html#wind-forcing)

Reply:
With the new definition of layer mass-thickness $h$, it is simply $\tau/h$.